kuma/core/tests/test_commands.py

```diff
@@ -0,0 +1,35 @@
+from django.core.management import call_command, CommandError
+from django.utils.six import StringIO
+
+from nose.tools import eq_
+
+from kuma.core.tests import TestCase
+from kuma.users.models import User
+from kuma.users.tests import user
+
+
+class TestIHavePowerCommand(TestCase):
+    def test_help(self):
+        out = StringIO()
+        with self.assertRaises(CommandError) as cm:
```

Comment:
I was just looking for something more descriptive for people that are not you :)